packages/toolkit/src/view/catalog/components/CreateCatalogDialog.tsx

```diff
@@ -19,10 +19,10 @@ import { EntitySelector, LoadingSpin } from "../../../components";
 import { InstillStore, useInstillStore, useShallow } from "../../../lib";
 import { useUserNamespaces } from "../../../lib/useUserNamespaces";
 
-const CreateCatalogFormSchema = z.object({
+export const CreateCatalogFormSchema = z.object({
   name: z.string().min(1, { message: "Name is required" }),
   description: z.string().optional(),
-  // tags: z.string().optional(),
+  tags: z.union([z.array(z.string()), z.string()]).optional(),
```

Comment:
@thewbuk I feel weird about this type, I think it can just be string[] and we can call the day?